tests/zfs-tests/tests/perf/regression/random_reads.ksh

```diff
@@ -51,49 +51,31 @@ function cleanup
 trap "log_fail \"Measure IO stats during random read load\"" SIGTERM
 log_onexit cleanup
 
-recreate_perf_pool
-populate_perf_filesystems
+for logtype in $PERF_LOG_TYPES; do
+	recreate_perf_pool $logtype
+	populate_perf_filesystems
 
-# Aim to fill the pool to 50% capacity while accounting for a 3x compressratio.
-export TOTAL_SIZE=$(($(get_prop avail $PERFPOOL) * 3 / 2))
-
-# Variables specific to this test for use by fio.
-export PERF_NTHREADS=${PERF_NTHREADS:-'16 32'}
-export PERF_NTHREADS_PER_FS=${PERF_NTHREADS_PER_FS:-'0'}
-export PERF_IOSIZES=${PERF_IOSIZES:-'8k'}
-export PERF_SYNC_TYPES=${PERF_SYNC_TYPES:-'1'}
-
-# Layout the files to be used by the read tests. Create as many files as the
-# largest number of threads. An fio run with fewer threads will use a subset
-# of the available files.
-export NUMJOBS=$(get_max $PERF_NTHREADS)
-export FILE_SIZE=$((TOTAL_SIZE / NUMJOBS))
-export DIRECTORY=$(get_directory)
-log_must fio $FIO_SCRIPTS/mkfiles.fio
-
-# Set up the scripts and output files that will log performance data.
-lun_list=$(pool_to_lun_list $PERFPOOL)
-log_note "Collecting backend IO stats with lun list $lun_list"
```

Comment:
Do we no longer need to collect the "backend IO stats" here (we still seem to do this for the other perf/regression tests)?

Reply:
Messed up the upstream merge here.